core/genesis_test.go

```diff
@@ -176,3 +176,16 @@ func TestSetupGenesis(t *testing.T) {
 		}
 	}
 }
+
+// TestRegistryInGenesis tests if RegistrySmartContract sits in the state of genesis block
+func TestRegistryInGenesis(t *testing.T) {
+	db := rawdb.NewMemoryDatabase()
+	MainnetGenesisBlock().MustCommit(db)
```

Comment:
FYI this check also holds true for our other genesis blocks.

Reply:
> FYI this check also holds true for our other genesis blocks.

Added Dev, Alfajores, Baklava. Thanks for pointing it out!